### PR TITLE
Document the Util module

### PR DESCRIPTION
**Pull request by jmid**
Here's a PR providing documentation for the shared `Util` module.

### CODE REVIEW

File: lib/util.ml [hunk 1]
```diff
@@ -1,5 +1,3 @@
-(* Repeat a non-determistic property                          *)
```
Comment (n-osborne) [inline lib/util.ml:1]: To be completely exact, this repeat a property, and it is useful when the property is non-deterministic.
Not sure it is that important though :-)

File: lib/util.mli [hunk 2]
```diff
@@ -0,0 +1,49 @@
+(** The Util module contains a number of reusable functions
+    handy for multicore testing. *)
+
+
+val repeat : int -> ('a -> bool) -> 'a -> bool
+(** [repeat num prop] iterates a (non-determistic) property [prop]
```
Comment (n-osborne) [inline lib/util.mli:6]: Same here.

**Review by n-osborne — APPROVED** [review]

File: lib/util.ml (hunk 1 above)
Comment (jmid) [reply lib/util.ml:1]: OK, point well-taken :+1:
This comment is removed by the PR, so it is in a sense already addressed :slightly_smiling_face:

File: lib/util.mli (hunk 2 above)
Comment (jmid) [reply lib/util.mli]: Good point :+1: 
I agree that the comment could be clearer. I've tried to rephrase it in commit 8e78a8c

**Pull request merged**